Assets/MixedRealityToolkit/_Core/Definitions/Handedness.cs

```diff
@@ -10,24 +10,28 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
     /// It is up to the developer to determine whether this affects the use of a controller or not.
     /// "Other" defines potential controllers that will offer a "third" hand, e.g. a full body tracking suit.
     /// </summary>
-    [Flags]
```

Comment:
How does this change affect classes like `ControllerFinder`?

Reply:
We'll need to re-evaluate controller finder and adapt when we get to that implementation.